docs/_docs/2_3_querying_data.md

```diff
@@ -148,5 +148,5 @@ Additionally, `HoodieReadClient` offers the following functionality using Hudi's
 
 ## Presto
 
-Presto is a popular query engine, providing interactive query performance. Presto currently supports only read optimized querying on Hudi tables. 
+Presto is a popular query engine, providing interactive query performance. Presto currently supports only read optimized queries on Hudi tables. 
```

Comment:
>Presto currently supports only read optimized queries

this is snapshot query right.. Can you please make another pass for any lines we missed like this

Reply:
> @bhasudha Thanks for undertaking this! Feels great to have help with docs :)
> 
> Please take a another read of all the pages, specifically questioning each mention of "Read optimized" and ask if we really mean "snapshot" ..
> 
> Otherwise LGTM

Sure. Let me check and update the PR

Reply:
> > Presto currently supports only read optimized queries
> 
> this is snapshot query right.. Can you please make another pass for any lines we missed like this

@vinothchandar I looked at this. This is my understanding. Please correct if I am wrong.
When we refer to "snapshot" it means  queries can see data committed as of that point in time. For COW table type, this refers to latest parquet files for each file group. For MOR table type snapshot queries can see real-time data merged from base Parquet file and delta log file (AVRO files). Wrt Presto specifically, we are not supporting reading from avro files yet (referring to https://issues.apache.org/jira/browse/HUDI-305) . So I kept this as ReadOptimized instead of Snapshot.  Later when we support this, we can switch to Snapshot query support in Presto. What do you think?

Reply:
+1 to keep this as ReadOptimized.